packages/superset-ui-chart-controls/src/types.ts

```diff
@@ -200,11 +197,7 @@ export interface BaseControlConfig<
   warning?: ReactNode;
   error?: ReactNode;
   // override control panel state props
-  mapStateToProps?: (
-    state: ControlPanelState,
-    control: this,
-    actions?: ControlPanelActionDispatchers,
```

Comment:
The `actions` argument was once provided, but has since removed after some refactoring.